Dev/teampage PRA-156

ElenaGor8 opened the pull request:
## One Line Description
Responsive Our Team page and footer

## Requirements

_What this code should do_

Our Team page shows 9 team members, their images and short "about" text, each member card is clickable to their personal github pages.
Footer should show in the bottom of the pages throughout the app.
footer icons click out to corresponding webpages.
footer nav buttons styled and properly routed.
Buttons changed to solid green in Homepage and Contact pages
Responsive design

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

See footer in laptop and mobile layouts, click on Meet the Team to see Our Team page, also check it in mobile version.

## UI/UX Outcome

_Attach a screenshot here_
![OurTeammobile](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/302d6015-108c-4284-b3a9-86c587911d42)

![OutTeam_Footer_mobile](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/5d45bd5e-8386-4184-849e-9bc10a461490)

![OurTeam827](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/27e6944c-f2b6-49e1-be9e-eae54144098f)

## Checklist


- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

AnnaPestova1 reviewed (approved):
We have the well behave footer now! Great! I like all here

AnnaSolovykh reviewed (approved):
LGTM!

Ekaterina-Bondareva commented on src/components/Layout/OurTeam.tsx:
```diff
@@ -1,14 +1,112 @@
+    role: "Back-end",
+    imageUrl: "/images/kitten.png",
+    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    githubUrl: "https://github.com/AigulYe"
```
Could you please fix the link. This link is not work. I suppose the correct one is: `https://github.com/AigulY`

Ekaterina-Bondareva reviewed (changes requested):
Great job! Love how the page looks like! Please fix one line of code (GitHub link).

rebekahcallkacz reviewed (commented):
Not sure if you want to address this in this PR, but the spacing is off on the Add Recipe page. The footer is slightly hidden on my screen.

rebekahcallkacz reviewed (commented):
I might use the term "Front End Developer" and "Back End Developer" (or Engineer instead of developer) instead of just "Front-end" and "Back-end". You usually put a person's "title" instead of just what area they're in.

You'll also want to italicize that title/role to separate it from names - see [this example](https://zelusanalytics.com/).

rebekahcallkacz reviewed (approved):
LGTM other than the link that Kate already pointed out.

ElenaGor8 commented:
Made changes to OurTeam page and Footer for responsiveness, added mentors' cards, adjusted height of the Add recipe page, corrected Aigul's gihub link

Ekaterina-Bondareva reviewed (approved):
LGTM! Good job!
